include/WPA/VersionedFlowSensitive.h

```diff
@@ -167,6 +186,8 @@ class VersionedFlowSensitive : public FlowSensitive
     double meldMappingTime;  ///< Time to map MeldVersions to Versions.
     double versionPropTime;  ///< Time to propagate versions to versions which rely on them.
     //@}
+
+    static VersionedFlowSensitive *vfspta;
```

Comment:
Do we need a null initialization in VersionedFlowSensitive.cpp?

Reply:
Yes, my apologies.